src/main.cpp

```diff
@@ -18,9 +18,17 @@ int main(int argc, char *argv[]){
         if(strcmp(argv[i], "-L") == 0){
             length = atoi(argv[i+1]);
         }
+
         if (std::string(argv[i]) == "-V"){
             std::cout << "passgen version: " << passgen_VERSION_MAJOR << "." << passgen_VERSION_MINOR  << "."<< passgen_VERSION_REVISION << "\n";
         }
+
+        if (std::string(argv[i]) == "-H"){
+			std::cout << "Use 'passgen -L 20' to generate a random password with default character set of length 20\n";
```

Comment:
This help message needs improvements. You're printing only examples. Before that we should print all the options and functions, followed by examples. 
See how help is printed for other commands and see the pattern. Maybe see manpages of basic commands like `ls`.